lib/providers/openstreetmap.js

```diff
@@ -175,4 +176,96 @@ var OpenStreetMap = Provider.extend(function (self, opts) {
     };
 });
 
+OpenStreetMap.fixture = function(opts) {
+    /**function:OpenStreetMap.fixture(opts)
+
+    Returns an HTTP resource fixture for an OpenStreetMap location query.
+
+    :param string opts.query:
+        The address that is to be queried. Required.
+
+    :param array_of_strings opts.address_list:
+         An array of the list of `formatted_address`'s that should be sent in
+         the response. If response_data is included, this will be ignored.
+         Defaults to ``[]``.
+
+    :param boolean hard_boundary:
+        Whether the request should limit results to a fixed bounding box.
+        Defaults to `true`.
+
+    :param integer address_limit:
+        The maximum number of search results that should be requested. Defaults
+        to `30`.
+
+    :param array bounding_box:
+        The bounding box to submit in the search request. Should be provided
+        in the form ``[left_edge, top_edge, right_edge, bottom_edge]``, e.g.
+        ``["-18.3273", "-33.7652", "18.937", "-34.3329"]`` for South Africa.
+        Defaults to ``["-180.0", "90.0", "180.0", "-90.0"]`` for worldwide
+        search.
+
+    :param string opts.request_url:
+        URL for the HTTP request. Defaults to
+        ``"http://open.mapquestapi.com/nominatim/v1/search.php"``.
+
+    :param array_of_objects opts.response_data:
```

Comment:
Do we maybe need a `:type:` thing for `array_of_objects`, or is `array_of_objects` something Sphinx understands?